plutarch-quickcheck.cabal

```diff
@@ -1,18 +1,17 @@
 cabal-version:      3.0
 name:               plutarch-quickcheck
-version:            1.0.0
-synopsis:           Helpers for property testing plutarch
+version:            2.0.0
+synopsis:           Quickcheck for Plutarch.
 description:
-  Contains several helpers for law checks of Plutarch type classes, as well as 
-  helpers for property test writing. Some orphan instances useful for Plutarch 
-  property writing are also included.
+  Bridge between QuickCheck and Plutarch. The interfaces provide
+  easy ways to construct properties about Plutarch functions.
 
 homepage:           https://github.com/Liqwid-Labs/plutarch-quickcheck
 bug-reports:        https://github.com/Liqwid-Labs/plutarch-quickcheck/issues
 license:            Apache-2.0
 license-file:       LICENSE
-author:             Koz Ross
-maintainer:         Koz Ross <koz@mlabs.city>
+author:             Seungheon Oh
+maintainer:         Seungheon Oh <seungheon.ooh@gmail.com>
```

Comment:
This should probably be a more official address, such as one provided by MLabs or Liqwid Labs.